### PR TITLE
chore: update pr templates and issues

### PR DESCRIPTION
**Pull request by JeremyFriesenGitHub**
Here is a potential look at a new pr template: 

# PR Requirements Checklist

- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

# PR Description

- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
   -  Deletes all old templates (for PRs and Issues)
   -  Creates new templates (for PRs and Issues)

- **What is the current behavior?** (You can also link to an open issue here)
   - #161 

- **What is the new behavior (if this is a feature change)?**
   - N/A

- **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
   - N/A

- **Other information**:
   - More issue templates to come!

## Summary by CodeRabbit

- **New Features**
	- Introduced new structured templates for bug reports, architectural decision records, feature requests, and general issues to enhance clarity and completeness.
	- Added a configuration update to allow blank issues and included contact links for user outreach.

- **Bug Fixes**
	- Removed outdated templates for various issue types, streamlining the reporting process.

- **Documentation**
	- Enhanced the existing pull request template with a checklist to improve submission quality and adherence to guidelines.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
+
+* **What is the current behavior?** (You can also link to an open issue here)
+
+* **What is the new behavior (if this is a feature change)?**
+
+- **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
+
+* **Other information**:
```
[the remark this answers is not in the document]
Comment (JeremyFriesenGitHub) [reply .github/pull_request_template.md:15]: where would i put the placeholder text for the form description?

**Comment by JeremyFriesenGitHub** [comment]
I think its ready, idk how the yml will look though because it doesn't really preview like markdown, but i think I'm pretty sure I've got it right ig. I used [this](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms) as a resource. Hasith also mentioned [Brave's](https://github.com/brave/brave-browser/tree/master/.github/ISSUE_TEMPLATE) issue templates as well. Lmk what needs to be done!

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Pull Request Template
+
+## PR Requirements Checklist
+
+- [ ] The commit message follows our guidelines
+- [ ] Tests for the changes have been added (for bug fixes/features)
+- [ ] Docs have been added / updated (for bug fixes / features)
+
+## PR Description
```
Comment (HasithDeAlwis) [inline .github/pull_request_template.md:9]: Thoughts on simplifying this to just an SS (if applicable) and something like
```
## Changes in the PR
Provide an itemized list of changes
```
I feel like there is a lot of repeating yourself here

Comment (HasithDeAlwis) [reply .github/pull_request_template.md:9]: To clarify, please keep the requirements section 👍

**Review by HasithDeAlwis — COMMENTED** [review]
The issue templates are all pretty similar, do you think it's a good idea to combine some similar ones? 

Also don't forget to ask, "What is the solution?" for something like an feature. For an ADR we want a bit more, maybe something like "What is the solution", "What are the pros and cons", "Other alternatives considered". We're looking for a bit more for that one.

**Comment by JeremyFriesenGitHub** [comment]
> The issue templates are all pretty similar, do you think it's a good idea to combine some similar ones?
> 
> Also don't forget to ask, "What is the solution?" for something like an feature. For an ADR we want a bit more, maybe something like "What is the solution", "What are the pros and cons", "Other alternatives considered". We're looking for a bit more for that one.

ok, ill change to be more specific for types.

File: .github/pull_request_template.md (hunk 2 above)
Comment (JeremyFriesenGitHub) [reply .github/pull_request_template.md:9]: > Thoughts on simplifying this to just an SS (if applicable) and something like
> 
> ```
> ## Changes in the PR
> Provide an itemized list of changes
> ```
> 
> I feel like there is a lot of repeating yourself here

ok, will do

File: .github/pull_request_template.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (JeremyFriesenGitHub) [reply .github/pull_request_template.md]: hello, i might need further assistance

**Comment by JeremyFriesenGitHub** [comment]
> Caution
> 
> ## Review failed
> The head commit changed during the review from [d5488c0](https://github.com/cuhacking/2025/commit/d5488c079505423230ac5605ed7cf2f19cca139d) to [1c41293](https://github.com/cuhacking/2025/commit/1c41293d50141aca021559d1cafd2f6eb8cdc679).
> 
> 📝 Walkthrough
> Thank you for using CodeRabbit. We offer it for free to the OSS community and would appreciate your support in helping us grow. If you find it useful, would you consider giving us a shout-out on your favorite social media?
> 
> ❤️ Share
> 🪧 Tips

test

**Review by HasithDeAlwis — APPROVED** [review]

**Pull request merged**